tests/gtest/aviftest_helpers.cc

```diff
@@ -15,13 +15,58 @@
 
 #include "avif/avif.h"
 #include "avif/avif_cxx.h"
-#include "avif/internal.h"
 #include "avifpng.h"
 #include "avifutil.h"
 
 namespace avif {
 namespace testutil {
 
+//------------------------------------------------------------------------------
+// Duplicated from internal.h
```

Comment:
Update this comment. Perhaps just say CopyImageSamples is a copy of avifImageCopySamples?